### PR TITLE
Appveyor tests to check opensim-models

### PR DESCRIPTION
**Pull request by jimmyDunne**
Adds appveyor tests to the opensim-models repo.

@chrisdembia I would like some help making this work on appveyor. Locally, I can identify which models aren't able to be loaded using OpenSim 4.0; but I am not sure how this works remotely.

### CODE REVIEW

File: test_models.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        if fnmatch(name, pattern):
+            osimpaths.append(os.path.join(path, name))
+            modelNames.append(name)
+
+for i in range(0 , len(osimpaths) - 1):
+    try:
+        model = om.Model(osimpaths[i])
+        s = model.initSystem()
+    except:
+        print("Oops, Model Failed: " + modelNames[i])
```
[the remark this answers is not in the document]
Comment (chrisdembia) [reply test_models.py]:
```
except Exception as e:
    print(e.message)
```

[the remark this answers is not in the document]
Comment (aseth1) [reply test_models.py]: Cool! That would be nice to add so we can see, for example, a failure to load a .vtp file and then fix it.

**Comment by chrisdembia** [comment]
AppVeyor is now set up correctly to load and call `initSystem()` on every `.osim` file in the repository. However, the test fails because not all models load successfully:

```
Oops, Model 'SoccerKickingModel.osim' failed:
std::exception in 'SimTK::State & OpenSim::Model::initSystem()': Assigned an invalid SimTK::MobilizedBodyIndex
	In Object 'pelvis_offset' of type PhysicalOffsetFrame.
	Thrown at PhysicalFrame.cpp:73 in setMobilizedBodyIndex().
```

I think we can merge this PR now, and once the models are fixed, the AppVeyor test will eventually pass.

**Comment by chrisdembia** [comment]
Thank you.

**Pull request merged**